config/exchanges/BE_FR-ARA.yaml

```diff
@@ -0,0 +1,2 @@
+parsers:
+  exchange: ECO2MIX.fetch_exchange
```

Comment:
@corradio do you have a local version with coordinates (and rotation) for this (and the other) exchange(s)? I noticed they are correctly located on the screenshot you shared.

Reply:
This file shouldn't be there (notice how the keys in the yaml file are not alphabetically sorted). I will delete them.
The correct file is `FR-ARA_BE.yaml`

Reply:
This one shouldn't be here. Auvergne is not adjacent to Belgium. Will clean up.